tools/pipelines/templates/build-npm-package.yml

```diff
@@ -280,6 +285,23 @@ stages:
               tagName: ${{ parameters.tagName }}
               interdependencyRange: ${{ parameters.interdependencyRange }}
 
+        # Generate rollups and update package.json types to alpha/beta
+        - ${{ if or(eq(variables.releaseAlphaBetaBuild, 'alpha'), eq(variables.releaseAlphaBetaBuild, 'beta')) }}:
```

Comment:
This whole section can be moved down to just before the "pack" step. You can run the whole pipeline as normal and _right
before_ packing you set the version and dependencies and update the types. Then the pack steps should be able to run as
normal. So I think you only need one check right before pack.